how-to-contribute.md

```diff
@@ -89,6 +91,31 @@ bundle install
 Congratulations, you are done with setting up the Ruby environment for the MYRIAD website on your machine!
 
 
+## Setup a Ruby environment **(Windows)** <a name="setup-a-ruby-environment-windows"></a>
+For Windows users, here is a quick guide to install Ruby environment. Please refer to [Jekyll on Windows](https://jekyllrb.com/docs/installation/windows/) for more information. 
+
+
+### Download and install Ruby + Devkit
+1. Grab the latest version of RubyInstaller from [here](https://rubyinstaller.org/downloads/).
+2. Opt for default installation. Dont forget to check the `ridk install` on the last stage of the installation wizard.
```

Comment:
Reading through your guide, those are really the only instructions that are different in Windows compared to Linux. The following section of installing the project's dependencies is the same as under Linux. So I would request that instead of creating a whole new section for setting up a ruby environment under Windows, you rather add a subsection for installing ruby specifically. The instructions after of how to install the project's dependencies can be shared by both versions, which will both be less confusing for new users and easier for us to maintain should we wish to modify them.

Reply:
Indeed, it seems clearer this way. @HangJung: if it's ok with you, I'll let you make the changes through another commit ?

Reply:
Changes made!